daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/processors/unparsers/ElementKindUnparsers.scala

```diff
@@ -54,25 +54,22 @@ class SequenceCombinatorUnparser(ctxt: ModelGroupRuntimeData, childUnparsers: Ve
 
   override def nom = "Sequence"
 
-  // Sequences of nothing (no initiator, no terminator, nothing at all) should
-  // have been optimized away
-  Assert.invariant(childUnparsers.length > 0)
-
   // Since some of the grammar terms might have folded away to EmptyGram,
   // the number of unparsers here may be different from the number of
   // children of the sequence group.
-  Assert.invariant(ctxt.groupMembers.length >= childUnparsers.length)
+  Assert.invariant(ctxt.groupMembers.length == childUnparsers.length)
```

Comment:
This doesn't match the above comment, and also seems odd. If the SequenceCombinatorUnparser is filtering out emptry grammars, then it seems reasonable that there might be childUnparsers than groupMembers. Or does this mean things aren't being filtered out now?

Reply:
I think that this was another case where Mike and I were taking a look at this and thought this was something that would cause an issue with this change.  I changed it back and it doesn't seem to cause any issues.

Reply:
If using == works, I would stick with that. The more restrictive an assert is, the better. I'm a little surprised it works, I would think something from groupMembers would get filtered out, but it's not completely unreasonable.

Reply:
Done